TaskService/TaskService.cs

```diff
@@ -130,6 +132,30 @@ public TaskService(string targetServer, string userName = null, string accountDo
 			EndInit();
 		}
 
+		/// <summary>Initializes a new instance of the <see cref="TaskService"/> class.</summary>
+		/// <param name="targetServer">
+		/// The name of the computer that you want to connect to. If the this parameter is empty, then this will connect to the local computer.
+		/// </param>
+		/// <param name="userName">
+		/// The user name that is used during the connection to the computer. If the user is not specified, then the current token is used.
+		/// </param>
+		/// <param name="accountDomain">The domain of the user specified in the <paramref name="userName"/> parameter.</param>
+		/// <param name="userSecurePassword">
+		/// The password that is used to connect to the computer as a SecureString. If the user name and securePassword are not specified, then the current token is used.
+		/// </param>
+		/// <param name="forceV1">If set to <c>true</c> force Task Scheduler 1.0 compatibility.</param>
+		public TaskService(string targetServer, string userName = null, string accountDomain = null, SecureString userSecurePassword = null, bool forceV1 = false)

```

Comment:
Please change to the following to disambiguate from other ctor with optional params:
```cs
public TaskService(string targetServer, [Optional] string userName, [Optional] string accountDomain, [Optional] SecureString userSecurePassword, bool forceV1 = false)
```

Reply:
Thanks! Done.